Don't let rpm complain about a missing /etc/magic.mgc file

--- a/build/rpmfc.c.orig	2013-07-12 12:16:40.000000000 +0000
+++ b/build/rpmfc.c	2013-07-12 12:17:15.000000000 +0000
@@ -1024,7 +1024,7 @@ static int initAttrs(rpmfc fc)
 
 rpmRC rpmfcClassify(rpmfc fc, ARGV_t argv, rpm_mode_t * fmode)
 {
-    int msflags = MAGIC_CHECK | MAGIC_COMPRESS | MAGIC_NO_CHECK_TOKENS;
+    int msflags = MAGIC_COMPRESS | MAGIC_NO_CHECK_TOKENS;
     magic_t ms = NULL;
     rpmRC rc = RPMRC_FAIL;
 
